### ISSUE TITLE
package POSTIN scriptlet failure

### ISSUE THREAD

**Issue by mkudlej** [Tendrl/monitoring-integration#21]
I've tried to install this component from package ```tendrl-monitoring-integration-1.0.0-1.el7.centos.noarch``` which is from Tendrl 1.5.0 and I see this error:
```
 Installing : tendrl-monitoring-integration-1.0.0-1.el7.centos.noarch                              44/44 
mv: cannot stat ‘/etc/grafana/grafana.ini.rpmnew’: No such file or directory
warning: %post(tendrl-monitoring-integration-1.0.0-1.el7.centos.noarch) scriptlet failed, exit status 1
Non-fatal POSTIN scriptlet failure in rpm package tendrl-monitoring-integration-1.0.0-1.el7.centos.noarch
POSTTRANS: Running script
```

**Comment by mbukatov** [comment]
## Reproducer

1) On clean CentOS 7 machine, enable repositories for grafana, epel and tendrl copr
2) Run `yum install tendrl-monitoring-integration`

## Details

Postinstall scriptlet should not touch rpmnew (or rpmold) files.

The current scriptlets are:

```
# rpm -q --scripts tendrl-monitoring-integration 
preinstall scriptlet (using /bin/sh):
if [ $1 -eq 1 ] ; then
    mv /etc/grafana/grafana.ini /etc/grafana/grafana.ini.orig
fi
postinstall scriptlet (using /bin/sh):
if [ $1 -eq 1 ] ; then
    mv /etc/carbon/carbon.conf /etc/carbon/carbon.conf.tendrl-monitoring-integration
    mv /etc/httpd/conf.d/graphite-web.conf /etc/httpd/conf.d/graphite-web.conf.tendrl-monitoring-integration
    ln -s /etc/tendrl/monitoring-integration/carbon.conf /etc/carbon/carbon.conf
    ln -s /etc/tendrl/monitoring-integration/graphite-web.conf /etc/httpd/conf.d/graphite-web.conf
    mv /etc/grafana/grafana.ini.rpmnew /etc/grafana/grafana.ini
fi
preuninstall scriptlet (using /bin/sh):
if [ "$1" = 0 ] ; then
    rm -fr etc/carbon/carbon.conf /etc/httpd/conf.d/graphite-web.conf > /dev/null 2>&1
    mv /etc/carbon/carbon.conf.tendrl-monitoring-integration /etc/carbon/carbon.conf
    mv /etc/httpd/conf.d/graphite-web.conf.tendrl-monitoring-integration /etc/httpd/conf.d/graphite-web.conf
    mv /etc/grafana/grafana.ini.orig /etc/grafana/grafana.ini
fi
postuninstall program: /bin/sh
```

**Comment by rishubhjain** [comment]
@mkudlej @mbukatov The issue has been addressed, Please verify
Now grafana.ini is copied to /etc/tendrl/monitioring-integration/grafana/ (https://github.com/Tendrl/monitoring-integration/blob/master/tendrl-monitoring-integration.spec#L59) instead of /etc/grafana/, So this issue is no more reproducable

**Comment by mbukatov** [comment]
Ok, the scriptlets no longer touch rpmnew or rpmold files:

```
$ rpm -q --scripts -p tendrl-monitoring-integration-1.5.2-20170921T072407.1e2a7e6.noarch.rpm 2> /dev/null | grep rpm
$
```

That said, we should be using `%attr` to specify access rights, including unix group directly, instead of running `chgrp` in a post script. But that's out of scope of this issue.

I consider this issue fixed since `tendrl-monitoring-integration-1.5.2-20170921T072407.1e2a7e6`.

**Comment by nthomas-redhat** [comment]
@rishubhjain , @TimothyAsirJeyasing , Please fix this issue

**Comment by rishubhjain** [comment]
@nthomas-redhat This is issue is already fixed, @mbukatov has mentioned it in his previous comment.
"I consider this issue fixed since tendrl-monitoring-integration-1.5.2-20170921T072407.1e2a7e6."
@mbukatov Please verify and close the issue

**Comment by nthomas-redhat** [comment]
@rishubhjain, Since @mbukatov confirmed the fix, I am closing the issue
